File: app/src/main/java/com/nuhkoca/myapplication/data/remote/video/Paging.java
package com.nuhkoca.myapplication.data.remote.video;

import androidx.annotation.NonNull;
import androidx.databinding.BaseObservable;
import androidx.databinding.Bindable;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import com.nuhkoca.myapplication.BR;

public class Paging extends BaseObservable {

  @Expose @SerializedName("next") private String next;

  @Bindable public String getNext() {
    return next;
  }

  public void setNext(String next) {
    this.next = next;
    notifyPropertyChanged(BR.next);
  }

  @NonNull @Override public String toString() {
    return "Paging{" + "next='" + next + '\'' + '}';
  }
}

File: app/src/main/java/com/nuhkoca/myapplication/data/remote/video/Paging.kt
package com.nuhkoca.myapplication.data.remote.video

import androidx.databinding.BaseObservable
import androidx.databinding.Bindable
import com.google.gson.annotations.SerializedName
import com.nuhkoca.myapplication.BR

data class Paging(@SerializedName("next") var _next: String) : BaseObservable() {

    var next: String
        @Bindable get() = _next
        set(value) {
            _next = value
            notifyPropertyChanged(BR.next)
        }
}